packages/cli-kit/src/public/node/node-package-manager.ts

```diff
@@ -199,7 +199,7 @@ export async function installNodeModules(options: InstallNodeModulesOptions): Pr
     stderr: options.stderr,
     signal: options.signal,
   }
-  let args = ['install']
+  let args = [options.packageManager === 'yarn' ? 'add' : 'install']
```

Comment:
I'm not sure this is the right spot for this. I'm 90% sure this is the installation method to install *all* packages which `yarn add` wouldn't accomplish.

Reply:
you mean like ...

1. if `yarn`
   1. if `options.arg` not empty, use `add`
   2. else empty `args` array
2. else `args = ['install']`

Reply:
I'm not sure this function was ever meant to be used to install specific packages. I think that was something that Hydrogen utilized because it was available. Instead of making this function both install dependencies and add packages I think it would be better to create a separate `addNodeModulePackage` function.

What do you think?

Or since there is already a `getPackageManager` exported function could Hydrogen build the command out itself?

Reply:
I don't understand what is the hold back for this change. Is there a fear of don't know how it is being used elsewhere? Is it suppose to be an internal only function? or is it that simply introducing `yarn` as a possible package manager complicate things?